OnlineLibrary.Common/DBEntities/User.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace OnlineLibrary.Common.DBEntities
+{
+    class User
+    {
+        public int Id { get; set; }
+
+        public string Name { get; set; }
+
+        public string Email { get; set; }
+
+        public int Age { get; set; }
```

Comment:
Is it good to store age in the database? What if today I'm 20, but tomorrow will be 21?